pgpy/types.py

```diff
@@ -53,6 +53,28 @@ class Armorable(six.with_metaclass(abc.ABCMeta)):
                     '={crc}\n' \
                     '-----END PGP {block_type}-----\n'
 
+    # the re.VERBOSE flag allows for:
+    #  - whitespace is ignored except when in a character class or escaped
+    #  - anything after a '#' that is not escaped or in a character class is ignored, allowing for comments
+    __armor_regex__ = re.compile(r"""# This capture group is optional because it will only be present in signed cleartext messages
```

Comment:
I like that you moved this to the class level. I think this should just be named `__armor_regex` so the name gets mangled to hide it from subclasses and not confused as a magic member instead